README.md

```diff
@@ -129,6 +129,7 @@ public class Person {
 > this component uses domino-ui
 
 ```java
+//Note place in *-client directory
```

Comment:
maybe specify "client project or package", since not all projects follow that archetype